spec/controllers/concerns/hyrax/works_controller_behavior_spec.rb

```diff
@@ -172,4 +172,26 @@
       it_behaves_like 'allows show access'
     end
   end
+
+  describe '#manifest' do
+    include_context 'with a logged in user'
+    xit 'resolves json' do
+      # When I attempt to get this working, I encounter the following error:
```

Comment:
probably worth testing this in `generic_works_controller_spec.rb` instead, rather than holding this feature up on in-progress infrastructure.